frb_rust/src/misc/into_into_dart.rs

```diff
@@ -152,6 +152,7 @@ mod chrono_impls {
 }
 
 #[cfg(test)]
+#[cfg(not(target_family = "wasm"))]
```

Comment:
nit: what about constraining the conditional compile to the function

```
#[cfg(test)]
mod tests {
    use crate::misc::into_into_dart::IntoIntoDart;

    #[cfg(not(target_family = "wasm"))]
    #[test]
    fn test_zero_copy_buffer() {
        let raw: allo_isolate::ZeroCopyBuffer<Vec<u8>> = allo_isolate::ZeroCopyBuffer(vec![10]);
        assert_eq!(raw.into_into_dart().0, vec![10]);
    }
}
```

Reply:
Ok, fine as well :)